app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -417,6 +421,14 @@ class ChatActivity :
 
     var callStarted = false
 
+    private val CURRENT_AUDIO_MESSAGE_KEY = "CURRENT_AUDIO_MESSAGE"
+    private val CURRENT_AUDIO_POSITION_KEY = "CURRENT_AUDIO_POSITION"
+    private val CURRENT_AUDIO_WAS_PLAYING_KEY = "CURRENT_AUDIO_PLAYING"
+    private var RESUME_AUDIO_TAG = "RESUME_AUDIO_TAG"
```

Comment:
lets move the upper 4 strings to companion object (at the end of the file) and make them const.

Reply:
thanks, I've done the modifications in the commit that I've pushed right now, hope this is the right way to proceed